src/components/authorPageComponents/Gallery/Gallery.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+import style from './Gallery.module.scss';
+
+class Gallery extends React.Component {
+    render() {
+        return(
+            <>
+            {/* this.props.photosArrayOfObjects - this is an array of objects! */}
+            <p>{this.props.photosArrayOfObjects[0].file.url}....$Gallery</p>
```

Comment:
Look at the Carousel bootstrap [https://react-bootstrap.github.io/components/carousel/](https://react-bootstrap.github.io/components/carousel/), i think we need use this